sympy/physics/mechanics/particle.py

```diff
@@ -39,56 +41,58 @@ class Particle:
     >>> # Or you could change these later
     >>> pa.mass = m
     >>> pa.point = po
+    >>> # If preferred you can also attach a frame to a Particle
+    >>> pa.add_frame()
+    >>> pa.frame
+    pa_frame
 
     """
+    point = _Body.masscenter
 
-    def __init__(self, name, point, mass):
-        if not isinstance(name, str):
-            raise TypeError('Supply a valid name.')
-        self._name = name
-        self.mass = mass
-        self.point = point
-        self.potential_energy = 0
-
-    def __str__(self):
-        return self._name
-
-    def __repr__(self):
-        return self.__str__()
+    def __init__(self, name, point=None, mass=None):
```

Comment:
Not sure if we would like to add a `frame` kwarg here. Disadvantage would be that the order would be different compared to `RigidBody` due to argument order backwards compatibility.